spec/lib/import/csv_parser_sections_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+require 'import/csv_parser'
+
+describe Import::CSVParser do
+  let(:input_file) { File.join(fixture_path, 'csv', 'import_sections', 'tempest_sections.csv') }
+  let(:parser) { described_class.new(input_file) }
+
+  before do
+    # Don't print status messages during specs
+    allow($stdout).to receive(:puts)
+  end
+
+  describe 'initializer' do
```

Comment:
You can delete this test because it is already tested here:
https://github.com/curationexperts/heliotrope/blob/master/spec/lib/import/csv_parser_spec.rb#L13-L17